samples/openthread/cli/README.rst

```diff
@@ -518,6 +519,17 @@ To test the Thread 1.2 and Thread 1.3 features, complete the following steps:
             addresses: [fdde:ad00:beef:0:e0fc:dc28:1d12:8c2]
          Done
 
+Power consumption measurements
+==============================
+
+The Thread CLI sample can be used to perform power consumption measurements for Sleepy End Devices.
+After building and flashing with :file:`overlay-low_power.conf` and :file:`low_power.overlay` the device will start regular operation with UART console enabled.
```

Comment:
```suggestion
After building and flashing with :file:`overlay-low_power.conf` and :file:`low_power.overlay`, the device will start regular operation with the UART console enabled.
```
Also add an empty line before this line. Better to separate the instructional part from the descriptive first sentence.